gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/publisher/HiveConvertPublisher.java

```diff
@@ -79,6 +88,15 @@
   private EventSubmitter eventSubmitter;
   private final FileSystem fs;
   private final HiveSourceWatermarker watermarker;
+  private final HiveMetastoreClientPool pool;
+
+  public static final String PARTITION_PARAMETERS_WHITELIST = "hive.conversion.partitionParameters.whitelist";
```

Comment:
After enabling config store I will add properties to workunit so it will still be ok